src/patternfly/components/Menu/menu.scss

```diff
@@ -489,298 +417,217 @@
-}
-
-// Search
-.#{$menu}__search {
-  padding-block-start: var(--#{$menu}__search--PaddingTop);
-  padding-block-end: var(--#{$menu}__search--PaddingBottom);
-  padding-inline-start: var(--#{$menu}__search--PaddingLeft);
-  padding-inline-end: var(--#{$menu}__search--PaddingRight);
+// - Menu list
+.#{$menu}__list :where(.#{$divider}:is(li)) {
+  margin-block-start: var(--#{$menu}__list--divider--MarginTop);
+  margin-block-end: var(--#{$menu}__list--divider--MarginBottom);
 }
 
-// List
-.#{$menu}__list {
-  @include pf-v5-hidden-visible(var(--#{$menu}__list--Display));
-
-  padding-block-start: var(--#{$menu}__list--PaddingTop);
-  padding-block-end: var(--#{$menu}__list--PaddingBottom);
-
-  > .#{$divider} {
-    margin-block-start: var(--#{$menu}__list--c-divider--MarginTop);
-    margin-block-end: var(--#{$menu}__list--c-divider--MarginBottom);
-  }
-}
-
-// List item
+// - Menu list item
 .#{$menu}__list-item {
-  @include pf-v5-hidden-visible(var(--#{$menu}__list-item--Display));
-
-  position: relative;
+  align-items: start;
+  min-width: 0; // allow list item to shrink for text overflow
   color: var(--#{$menu}__list-item--Color);
   background-color: var(--#{$menu}__list-item--BackgroundColor);
 
-  &:hover {
+  &:has(.#{$menu}__item:hover, .#{$menu}__item:focus) {
     --#{$menu}__list-item--BackgroundColor: var(--#{$menu}__list-item--hover--BackgroundColor);
-    --#{$menu}__list-item--Color: var(--#{$menu}__list-item--hover--Color, inherit);
-  }
-
-  &:focus-within,
-  &.pf-m-focus {
-    --#{$menu}__list-item--BackgroundColor: var(--#{$menu}__list-item--focus-within--BackgroundColor);
-    --#{$menu}__list-item--Color: var(--#{$menu}__list-item--focus-within--Color, inherit);
-  }
-
-  &.pf-m-disabled,
-  &.pf-m-aria-disabled {
-    --#{$menu}__list-item--hover--BackgroundColor: transparent;
-    --#{$menu}__item--Color: var(--#{$menu}__list-item--m-disabled__item--Color);
-    --#{$menu}__item-toggle-icon: var(--#{$menu}__list-item--m-disabled__item-toggle-icon--Color);
-  }
-
-  &.pf-m-disabled {
-    --#{$menu}__list-item--focus-within--BackgroundColor: transparent;
-
-    pointer-events: none;
-  }
 
-  &.pf-m-aria-disabled .#{$menu}__item {
-    cursor: default;
+    .#{$menu}__item-external-icon {
+      color: var(--#{$menu}__item-external--Color);
+      opacity: 1;
+    }
   }
 
+  // - Menu item load
   &.pf-m-load {
-    --#{$menu}__list-item--hover--BackgroundColor: transparent;
-    --#{$menu}__list-item--focus-within--BackgroundColor: transparent;
     --#{$menu}__item--Color: var(--#{$menu}__list-item--m-load__item--Color);
   }
 
+  // - Menu item loading
   &.pf-m-loading {
-    --#{$menu}__list-item--hover--BackgroundColor: transparent;
-    --#{$menu}__list-item--focus-within--BackgroundColor: transparent;
-
     justify-content: center;
-    padding-block-start: var(--#{$menu}__list-item--m-loading--PaddingTop);
     overflow: hidden; // prevents spinner rotation from overflowing
+    pointer-events: none;
```

Comment:
@mcoker If you are referring to the spinner, it is currently a span with not `onClick` callback.  At this pont I would not anticipate a use case where we would click on it.

I believe the `tabindex="-1"` is correct.  We do not typically tab through our menu items.  We have been navigating them with the up and down arrow keys.  @thatblindgeye please correct me if I am wrong.

Reply:
That's correct regarding the tabindex. We do update the tabindex to 0 in React when an item actually receives focus via the arrow keys (just so the previously focused item will receive focus again upon entering the Menu, unless the menu was closed), but otherwise should be -1.

The/one of the only times we'd need to tab to content in a Menu is if there's actions in a footer/outside the actual menu item list.

Reply:
> If you are referring to the spinner, it is currently a span with not onClick callback. At this pont I would not anticipate a use case where we would click on it.

@tlabaj I'm referring to the `button.pf-v5-c-menu__item` that wraps it - you can see it in the screenshot below

<img width="740" alt="Screenshot 2024-02-05 at 3 40 31 PM" src="https://github.com/patternfly/patternfly/assets/35148959/6e5ad6e5-53ce-46dc-aa03-f56a85ca2c8a">

Reply:
@mcoker I see. Even thought the spinner is rendered in a button, the `onClick` is only called when the "View more" button is clicked and not the spinner.  If we wanted to allow for that we would need to modify the code.  I would not consider ding that right now unless we had a request for it.  At that point I would probably get UX's opinion on the interaction.